### DISCUSSION TITLE
Horizontal scroll pane

### DISCUSSION THREAD

**Discussion by artmoskvin** [zed-industries/zed#8143] [Feature Requests]
Hi! I'm probably missing something but I can't see any horizontal scroll pane in any tab. I tried looking for any related setting but found nothing.

**Comment by Moshyfawn** [comment]
There isn't one in Zed at the moment, if you're referring to the horizontal scrollbar #4427

**Reply by artmoskvin** [reply]
Sorry I meant scrollbar of course 🤦

**Reply by Navknight** [reply]
So how do we scroll horizontally? Shift + scroll also doesnt seem to work right now
